snap/snapcraft.yaml

```diff
@@ -65,15 +74,14 @@ parts:
     build-attributes:
       - enable-patchelf
     prime:
-      - "usr/lib/git-core"
-      - "usr/share/git-core"
-      - "usr/lib/x86_64-linux-gnu/libcurl-gnutls*"
-      - "usr/lib/x86_64-linux-gnu/libpsl*"
-      - "usr/lib/x86_64-linux-gnu/librtmp*"
-      - "usr/lib/x86_64-linux-gnu/libnghttp2*"
-      - "usr/lib/x86_64-linux-gnu/libldap*"
-      - "usr/lib/x86_64-linux-gnu/liblber*"
-      - "usr/lib/x86_64-linux-gnu/libsasl2*"
+      - "-usr/bin"
+      - "-usr/share/doc"
+      - "-usr/share/man"
+      # perl is part of the core22 / core24
+      - "-usr/share/perl"
+      - "-usr/lib/x86_64-linux-gnu/perl"
+      - "-usr/lib/x86_64-linux-gnu/libperl*"
+      - "-usr/lib/x86_64-linux-gnu/libgdbm*"
```

Comment:
Previous version has some missing library for `core24` ?

Reply:
yes, it was complaining about `libssh.so.0.4`